lib/css/components/_stacks-modals.less

```diff
@@ -68,15 +67,12 @@
         transform: translate3d(0, 0, 0) scale3d(1, 1, 1)
     }
 
-    //  IE11 and below don't correctly position the dialog box, so we need to set some additional
-    //  values to make sure it visibly centers.
+    //  IE11 and below aligns our dialog on subpixels causing a blurry layout. Let's just remove those transforms for IE11.
     @media screen and (-ms-high-contrast: active), (-ms-high-contrast: none) {
```

Comment:
Turns out that this isn't a good browser hack for IE11. It usually seems to work because `-ms-high-contrast: none` is no longer supported in Edge, but `active` is still one of the legal values there.

I would do this the other way around: Make `transform: none` the default, and only add actual transforms inside a "is not IE" check. The latter is pretty reliably doable by using a `@supports` feature query &ndash; and it doesn't matter which one. because IE doesn't support `@support` at all (yo dawg).

[Here's an example](https://github.com/StackExchange/Stacks/blob/26681d2bfa80293395802a733b79717482f5a4df/lib/css/components/_stacks-spinner.less#L29).

Reply:
So I just played with this and it appears that the blurriness doesn't even happen if you use 2d transforms instead of 3d transforms. Since we're not actually doing anything with the third dimension anyway, we should just do that for the best of both worlds, giving IE users transtions *and* crisp fonts.

Reply:
The first thing I tried was switching from translate3d, but when that second `position: fixed` was there, this was still blurry. Removing the second `position: fixed` in 
https://github.com/StackExchange/Stacks/pull/256/commits/aa9d212c939b0ccf6914f164b570514464a642d4 lets us have a nice and crispy `translate`. Nice!